GameEngineTests/Players/EpsilonGreedyPlayerTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Linq;
+using GameEngine;
+using GameEngine.Players;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace GameEngineTests.Players
+{
+    [TestClass]
+    public class EpsilonGreedyPlayerTests
+    {
+        private CardType[] OneCardOfEachColor
+        {
+            get
+            {
+                return new Deck(1).Cards
+                    .Where(card => card != CardType.Sun)
+                    .ToArray();
+            }
+        }
+
+        [TestMethod]
+        public void ShouldPlayCardOnOnlyOwlThatGoesFurthestWhenUsingGreedyStrategy()
+        {
+            var player = new EpsilonGreedyPlayer(1);
+            var board = new GameBoard(2, 1);
+            player.Hand.Add(OneCardOfEachColor);
+
+            var play = player.FormulatePlay(board);
+
+            Assert.AreEqual(CardType.Red, play.Card);
```

Comment:
💡 I don't think it's immediately clear why Red is the right card to expect. Is there a way to make that clearer?

What do you think about making a Greedy Player and testing its play on the same board state?